CMakeLists.txt

```diff
@@ -14,7 +14,9 @@ option(ENABLE_DOCUMENTATION "Build documentation." OFF)
 option(ENABLE_SPHINX_API_DOCS "Link Doxygen API docs to Sphinx" OFF)
 option(ENABLE_WIND_UTILS "Build wind utils along with Nalu-Wind" OFF)
 option(ENABLE_FFTW "Use the FFTW library to support ABLTopBC" OFF)
-option(ENABLE_HYPRE "Use HYPRE Solver library" OFF)
+option(ENABLE_HYPRE "Use HYPRE Solver library" ON)
```

Comment:
I'm not sure I meant to push this change that turns hypre on by default. Should I turn this back off?

Reply:
No we want it on by default now

Reply:
I would make it on by default. The trilinos solvers just aren't getting the same level of attention.